### PR TITLE
199 rework timeseries creation to contain species with observation after 1950

### LINKED ISSUE #199: Rework timeseries creation to contain species with observation after 1950
Currently species introduced prior to 1950 are excluded from the GAM timeseries this should be reworked to match option 2 of https://github.com/inbo/alien-species-portal/pull/74#issuecomment-2191167596. ea species should be included in the timeseries if they have at least 1 observation after 1950 independent of their year of introduction. This means the logic discribed here should be reworked.

https://github.com/inbo/aspbo/blob/ea385cec983aeddadf47cd7785112ad4c4c53730/src/05_occurrence_indicators_preprocessing.Rmd#L374

### PR DESCRIPTION
**Pull request by soriadelva**
null

### CODE REVIEW

**Comment by SanderDevisscher** [comment]
success 💪 🥳

**Comment by soriadelva** [comment]
@SanderDevisscher deze PR mag gereviewed worden, nu de flow weer werkt :)

File: src/05_occurrence_indicators_preprocessing.Rmd
Comment (SanderDevisscher) [inline src/05_occurrence_indicators_preprocessing.Rmd:1]: rename to cube_preprocessing.Rmd

File: src/05_occurrence_indicators_preprocessing.Rmd [hunk 1]
```diff
@@ -368,15 +368,15 @@ df_bl_xy %>% head()
 
 ## Apply temporal and spatial restraints
 
-Select alien species introduced after 1950 (`year_of_introduction`: 1950) or whose date of introduction is not known:
+Select alien species that were observed after 1950 (`year_cutoff`: 1950) or whose date of introduction is not known:
 
 ```{r select_aliens_after_year_of_introduction}
```
Comment (SanderDevisscher) [inline src/05_occurrence_indicators_preprocessing.Rmd]: select aliens with recent observations?

File: src/05_occurrence_indicators_preprocessing.Rmd [hunk 2]
```diff
@@ -395,18 +395,18 @@ recent_alien_species <-
...
+Number of taxa last observed since 1950 or whose introduction year is not specified:
 
 ```{r n_recent_taxa}
 nrow(recent_alien_species)
 ```
 
-List of alien species excluded as first introduction date is earlier than 1950:
+List of alien species excluded as last observation date is earlier than 1950:
 
 ```{r recent_alien_species}
```
Comment (SanderDevisscher) [inline src/05_occurrence_indicators_preprocessing.Rmd]: extinct_alien_species ?

File: src/05_occurrence_indicators_preprocessing.Rmd [hunk 3]
```diff
@@ -430,24 +430,24 @@ old_introductions_taxa <-
 old_introductions_taxa
 ```
 
-We save them in file `taxa_introduced_in_BE_before_1950.tsv` in  `data/output`:
+We save them in file `taxa_last_observed_in_BE_before_1950.tsv` in  `data/output`:
 
 ```{r save_old_introduced_taxa_as_tsv_file}
```
Comment (SanderDevisscher) [inline src/05_occurrence_indicators_preprocessing.Rmd]: save_last_observed_before ?

**Review by SanderDevisscher — CHANGES_REQUESTED** [review]
Since we changed the name of this script to cube_preprocessing (see #201) it is best to do the following (in order): 

- [x] rename 05_occurrence_indicators_preprocessing.Rmd to cube_preprocessing on this branch
- [x] do a rebase to make sure other changes in cube_preprocessing are included
- [x] solve possible conflicts
- [ ] solve other issues

**Comment by soriadelva** [comment]
Rebase with uat?

**Comment by SanderDevisscher** [comment]
> Rebase with uat?

yes

**Comment by soriadelva** [comment]
> > Rebase with uat?
> 
> yes

I got the following error: A lock file already exists in the repository, which blocks this operation from completing. any ideas?

**Review by SanderDevisscher — APPROVED** [review]

**Pull request merged**